Sources/Instrumentation/Tracing/Span.swift

```diff
@@ -85,23 +84,23 @@ extension Span {
 // ==== ----------------------------------------------------------------------------------------------------------------
 // MARK: Span Event
 
-/// An event that occured during a `Span`.
+/// An event that occurred during a `Span`.
 public struct SpanEvent {
     /// The human-readable name of this `SpanEvent`.
     public let name: String
 
     /// One or more `SpanAttribute`s with the same restrictions as defined for `Span` attributes.
     public var attributes: SpanAttributes
 
-    /// The `DispatchTime` at which this event occured.
-    public let timestamp: DispatchTime
+    /// The `DispatchTime` at which this event occurred.
+    public let timestamp: Timestamp
 
     /// Create a new `SpanEvent`.
     /// - Parameters:
     ///   - name: The human-readable name of this event.
-    ///   - attributes: The `SpanAttributes` describing this event. Defaults to no attributes.
-    ///   - timestamp: The `DispatchTime` at which this event occured. Defaults to `.now()`.
-    public init(name: String, attributes: SpanAttributes = [:], at timestamp: DispatchTime = .now()) {
+    ///   - attributes: attributes describing this event. Defaults to no attributes.
```

Comment:
Can we add the "The"s back in so it's consistent with the rest? Also happy to change everything else to this style 😊